.github/workflows/ci.yaml

```
@@ -43,5 +43,6 @@ jobs:
         shell: bash
         env:
           PATH_TO_GITBOOK_DOCS: ${{ github.workspace }}/apps/nextjs-website/docs
+          COOKIE_DOMAIN_SCRIPT: ${{ github.workspace }}/apps/nextjs-website/docs
```

```suggestion
          COOKIE_DOMAIN_SCRIPT: ${{ secrets.COOKIE_DOMAIN_SCRIPT }}
```

We have set a secret called `COOKIE_DOMAIN_SCRIPT` on every environment, so the workflow will get that value there.